### ISSUE TITLE
Null warning not reported for non-nullable parameter with null default value

### ISSUE THREAD

**Issue by cston** [dotnet/roslyn#25346]
Warning should be reported for `= null`.
```c#
class C
{
    static void F(object o = null)
    {
    }
}
```

**Comment by cston** [comment]
cc @NickCraver

**Comment by cston** [comment]
Fixed in https://github.com/dotnet/roslyn/pull/26726.
